src/Domains/Social/Messages/Observers/MessageObserver.php

```diff
@@ -19,6 +20,14 @@ public function creating(Message $message)
                 getChildrenCount: true
             ))->execute();
         }
+
+        if ($message->app->get('validate-message-schema')) {
+            $checkJson = new ValidateMessageSchemaAction($message, $message->messageType);
+            $errors = $checkJson->execute();
+            if (! empty($errors)) {
+                throw new ValidationException(implode(', ', $errors));
```

Comment:
@rwhite27 in this case create a new message validation exception to be more specific